index/src/graph/mod.rs

```diff
@@ -16,6 +16,7 @@ pub use self::{
     dex::{Dex, DexIndexer, DexPair},
 };
 
+// TODO: having 2 graphs is pretty confusing
```

Comment:
Do you have any idea to fix the problem?

Reply:
the registry graph needs to be transformed into another internal graph, which is certain and inevitable, what is to improve is the naming, even those who are involved in the coding still need time to distinguish one from another, but I don't intend to do that in this PR